packages/core/src/SkipMenu/SkipMenu.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from '../Flex'
+import { Link } from '../Link'
+import { SrOnly } from '../SrOnly'
+
+const OffScreenPanel = styled(SrOnly)`
+  &:focus-within {
+    height: auto;
+    width: 100%;
+    max-width: 100%;
+    clip: auto;
+    margin: 0;
+    overflow: unset;
+  }
+`
+
+const SkipMenu = ({ className, skipLinks, ...props }) => {
+  if (!skipLinks?.length) return null
+  return (
+    <OffScreenPanel
+      data-testid='skip-menu'
+      as={Flex}
+      className={className}
+      {...props}
+    >
+      {skipLinks?.map(({ label, targetId, ...skipLink }) => (
+        <Link
+          key={`skipLink_${targetId}`}
+          href={`#${targetId}`}
```

Comment:
n-b maybe as a future enhancement we log an error if the `targetId` provided is already prefixed with `#`, or we could handle it gracefully in the component.